### DISCUSSION TITLE
How to provide port for the external DB?

### DISCUSSION THREAD

**Discussion by pratirvce** [fossology/fossology#2071] [General]
I am trying to use the web and scheduler containers to use an external postgres DB but that database exposes the port 5433 rather than 5432. The containers are trying to reach the DB on 5432 by default. How should I explicitly provide the port for the db in docker-compose.yml?

**Comment by GMishx** [comment]
Try adding it to `FOSSOLOGY_DB_HOST` like `FOSSOLOGY_DB_HOST=db:5433`
